rxbinding/src/main/java/com/jakewharton/rxbinding2/internal/Preconditions.java

```diff
@@ -27,6 +30,15 @@ public static void checkArgument(boolean assertion, String message) {
     return value;
   }
 
+  public static boolean isNotOnMainThread(Observer<?> observer) {
```

Comment:
Definitely not two methods. How about just `checkMainThread` with a boolean not placed at each call site to negate the result.